examples/header-based-routing/header_based_routing.bal

```diff
@@ -37,7 +35,7 @@ service /hbr on new http:Listener(9090) {
             return;
         }
         //[getHeader()](https://ballerina.io/swan-lake/learn/api-docs/ballerina/#/ballerina/http/latest/http/classes/Request#getHeader) returns header value of the specified header name.
-        string nameString = req.getHeader("x-type");
+        string nameString = checkpanic req.getHeader("x-type");
```

Comment:
shall we handle the error here rather panicing?

Reply:
Fixed